--- libchipcard-5.0.4/src/tools/cardcommander/cardcommander.cpp.orig	2018-10-02 16:58:36.755521643 +0100
+++ libchipcard-5.0.4/src/tools/cardcommander/cardcommander.cpp	2018-10-02 16:58:59.126944028 +0100
@@ -45,7 +45,7 @@
 using namespace std;
 
 #define k_PRG_VERSION_INFO \
-  "cardcommander v0.4  (part of libchipcard v"k_CHIPCARD_VERSION_STRING")\n"\
+  "cardcommander v0.4  (part of libchipcard v" k_CHIPCARD_VERSION_STRING ")\n"\
   "(c) 2006 Martin Preuss<martin@libchipcard.de>\n" \
   "This program is free software licensed under GPL.\n"\
   "See COPYING for details.\n"
